src/QsCompiler/QsTargeting/Targeting.fsproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="CapabilityLeveler.fs" />
+    <Compile Include="CallGraphWalker.fs" />
+    <Compile Include="Target.fs" />
```

Comment:
Seems like having this here is causing the build issue. This file was removed, right?

Reply:
And of course it built fine on my machine because I didn't actually delete the file, I just removed it from git... Fixed now, I think.
Thanks for the catch!